chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java

```diff
@@ -277,39 +275,9 @@ private void fillTestSummaryAndScalabilityData(String sessionId) {
 
             dataForScalabilityPlots.put(testEntity, new HashMap<String, Double>());
 
-            //??? todo JFG_777 - decide how to provide session comparison and decision making for back compatibility
-            // workaround for back compatibility
-            // create dummy workloadTaskData entity for decision maker
-            WorkloadTaskData workloadTaskData = new WorkloadTaskData();
-            Map<MetricEntity,MetricSummaryValueEntity> metricsForThisTest = standardMetricsMap.get(testEntity);
-            for (MetricEntity metricEntity : metricsForThisTest.keySet()) {
-                if (StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.THROUGHPUT_ID).contains(metricEntity.getMetricId())) {
-                    workloadTaskData.setThroughput(new BigDecimal(metricsForThisTest.get(metricEntity).getValue()));
-                    dataForScalabilityPlots.get(testEntity).put(StandardMetricsNamesUtil.THROUGHPUT_ID, metricsForThisTest.get(metricEntity).getValue());
-                }
-                if (StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.SUCCESS_RATE_ID).contains(metricEntity.getMetricId())) {
-                    workloadTaskData.setSuccessRate(new BigDecimal(metricsForThisTest.get(metricEntity).getValue()));
-                }
-                if (StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.LATENCY_ID).contains(metricEntity.getMetricId())) {
-                    workloadTaskData.setAvgLatency(new BigDecimal(metricsForThisTest.get(metricEntity).getValue()));
-                    dataForScalabilityPlots.get(testEntity).put(StandardMetricsNamesUtil.LATENCY_ID, metricsForThisTest.get(metricEntity).getValue());
-                }
-                if (StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.LATENCY_STD_DEV_ID).contains(metricEntity.getMetricId())) {
-                    workloadTaskData.setStdDevLatency(new BigDecimal(metricsForThisTest.get(metricEntity).getValue()));
-                    dataForScalabilityPlots.get(testEntity).put(StandardMetricsNamesUtil.LATENCY_STD_DEV_ID, metricsForThisTest.get(metricEntity).getValue());
-                }
-            }
-
             String testStatusComment = "";
             Decision testStatus = Decision.OK;
 
-            // Success rate
-            Decision testSuccessRateStatus = decisionMaker.decideOnTest(workloadTaskData);
```

Comment:
as a result com.griddynamics.jagger.engine.e1.reporting.DefaultSessionStatusDecisionMaker and com.griddynamics.jagger.engine.e1.reporting.SessionStatusDecisionMaker are not needed any more